### DISCUSSION TITLE
How to hide inbox for agents on chatwoot

### DISCUSSION THREAD

**Discussion by klebinhopk** [chatwoot/chatwoot#8863] [General]
![image](https://github.com/chatwoot/chatwoot/assets/16194393/5d9eca96-a3c3-4dff-b99b-656a9ed95ade)

I would like to know if there is any option to hide the inbox for agents.
This way, agents would only be able to see conversations that were assigned to them in the Teams tab.
Is there any such setting in chatwoot?
I couldn't find it.

**Comment by Evoltstore** [comment]
I would like to know this as well, 
view only assigned messages or team messages, but **have no access to the full inbox.** 

Is this possible to achieve?

**Comment by jcmatoskx** [comment]
Seems there is a request for this already https://github.com/chatwoot/chatwoot/issues/4216
and some updates were released but I don't think they are working as described https://www.chatwoot.com/blog/team-permissions

**Comment by vishnu-narayanan** [comment]
This is being actively worked on and should be available within the next couple of releases. 

ref: https://github.com/chatwoot/chatwoot/issues/4216#issuecomment-2128159545
